README.md

```diff
@@ -51,7 +51,13 @@ Some of *SwarmKit*'s main features are:
 - **Scheduling**
   - **Resource Awareness**: *SwarmKit* is aware of resources available on nodes and will place tasks accordingly.
   - **Constraints**: Operators can limit the set of nodes where a task can be scheduled by defining constraint expressions.
-    Constraints can match various node attributes such as IDs, names and labels (e.g. `node.labels.foo!=bar1`).
+    Multiple constraints find nodes that satisfy every expression. Constraint can match node attributes including:
```

Comment:
Let's also say it's an AND match - I think that makes it clearer.

s/Constraint/Constraints/